Formula/lilypond.rb

```diff
@@ -0,0 +1,98 @@
+class Lilypond < Formula
+  desc "Music notation for everyone"
+  homepage "https://lilypond.org"
+  url "https://lilypond.org/download/sources/v2.22/lilypond-2.22.1.tar.gz"
+  sha256 "72ac2d54c310c3141c0b782d4e0bef9002d5519cf46632759b1f03ef6969cc30"
+  license "GPL-3.0-or-later"
+
+  depends_on "autoconf" => :build
+  depends_on "fontforge" => :build
+  depends_on "gettext" => :build
+  depends_on "glib" => :build
+  depends_on "imagemagick" => :build
+  depends_on "pkg-config" => :build
+  depends_on "t1utils" => :build
+  depends_on "texi2html" => :build
+
+  depends_on "fontconfig"
+  depends_on "freetype"
+  depends_on "ghostscript"
+  depends_on "guile@2"
+  depends_on :macos
+  depends_on "pango"
+  depends_on "python@3.9"
+
+  uses_from_macos "flex" => :build
+  uses_from_macos "m4" => :build
+
+  resource "bison" do
+    url "https://ftp.gnu.org/gnu/bison/bison-3.7.6.tar.xz"
+    sha256 "67d68ce1e22192050525643fc0a7a22297576682bef6a5c51446903f5aeef3cf"
+  end
```

Comment:
This should use the `bison` formula.

Reply:
same as above (I cannot use bison provided by macOS)

Reply:
Tried to add `uses_from_macs` but got a lot of errors during compilation.